docs/reference/target-frameworks.md

```diff
@@ -73,6 +73,7 @@ Universal Windows Platform | uap | uap [uap10.0] |
 | | | netstandard1.5 |
 | | | netstandard1.6 |
 | | | netstandard2.0 |
+| | | netstandard2.1 |
```

Comment:
@vasont I am curious how did you find out that `netstandard2.1` is a supported framework. Did you look at the Framework definition in [FrameworkConstants.cs](https://github.com/NuGet/NuGet.Client/blob/64f2febf7d1de8b0228eca5afed97a96c5a30bba/src/NuGet.Core/NuGet.Frameworks/FrameworkConstants.cs#L161) on the dev branch ?

Reply:
Unless there's a new mapping NuGet does not have define constants for newer tfms.

So we know that .NET standard 2.1 exists because of https://devblogs.microsoft.com/dotnet/announcing-net-standard-2-1/

Reply:
Is it the same case for `netstandard1.7` in [FrameworkConstants.cs](https://github.com/NuGet/NuGet.Client/blob/64f2febf7d1de8b0228eca5afed97a96c5a30bba/src/NuGet.Core/NuGet.Frameworks/FrameworkConstants.cs#L158) ?

Reply:
.NET Standard 1.7 never really shipped. 
NuGet does not really try to enforce the existence tooling for targeting certain framework.
The difference is ok, because doing so would be too tedious.